src/views/Maintenancepage.vue

```diff
@@ -0,0 +1,81 @@
+            />
+            <div class="maintenance-page_description level is-mobile">
+                <div class="maintenance-page_title">
+                    {{ $t('views.maintenance.title') }}
+                </div>
+                <div class="maintenance-page_subtitle">
+                    {{ $t('views.maintenance.subtitle') }}
+                </div>
+            </div>
+        </div>
+    </main>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import mainStore from '@/store/main-store/MainStore';
+@Component({
+    name: 'Maintenance',
+    components: {},
+})
+export default class Maintenance extends Vue {
+    private mainStore = mainStore.context(this.$store);
+
+    private get currentLanguage(): string {
+        return this.mainStore.state.currentLanguage;
+    }
+}
+</script>
+
+<style lang="scss" scoped>
+.maintenance-page {
+    height: 100vh;
+    background-color: #ffffff;
+    &_content {
+        flex-direction: column;
+        justify-content: space-around;
+        height: 800px;
+        padding: 20px;
+    }
+    &_img {
+        max-width: 650px;
+    }
+    &_description {
+        flex-direction: column;
+        height: 120px;
+    }
+    &_title {
+        font-weight: bold;
+        font-size: 50px;
+        color: #3a4750;
+    }
+    &_subtitle {
+        font-size: 30px;
+    }
+}
+@media (max-width: 750px) {
```

Comment:
@gonzalo-plaza We have tree diferents mixins to use this media, you can use touch, mobile and desktop, like that Jose is using in appoinment maintenance page